Add missing assembly to 'Creating the Data Set' section in the 'Working with linq' tutorial

KanagawaMarcos opened the pull request:
## Summary

The Creating the Data Set section was missing two assembly that it needs in order to compile. If the the user tried to compile the code as the tutorial told him to do, he would see an error message, instead of what was prompted in the image used at the end of this section.

KanagawaMarcos committed:
Add missing assembly to 'Creating the Data Set'

The Creating the Data Set section was missing two assembly that it needs in order to compile. If the the user tried to compile the code as the tutorial told him to do, he would see an error message, instead of what was prompted in the image used at the end of this section.

rpetrusha reviewed (changes requested):
Thanks, @Marcos-Costa, for contributing to the dotnet/docs repo. This is a useful addition, though it would be a good idea to add `using System;` to the list of imported namespaces as well.

Marcos-Costa commented on docs/csharp/tutorials/working-with-linq.md:
````diff
@@ -83,6 +83,13 @@ static IEnumerable<string> Ranks()
+For this to compile you’ll need to add the following two lines to the top of the file:
+
+```csharp
+using System.Collections.Generic;
````
Thanks for this addition. I agree with the fact that assumptions like that may bring some problems to newcomers.

KanagawaMarcos committed:
Add System namespace to Creating the Data Set

For this section of the tutorial compile, the user must have those three assembly; System, Sytem.Collection.Generic and System.Linq. Before this commit, only two of them were in the tutorial. The Third using statement was assumpted to be added by the SDK. Now the tutorial ensures that the user must add all necessary namespaces at the top of the file.

rpetrusha commented:
Thanks, @Marcos-Costa, for making the additional change. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.
